src/core/runner.vala

```diff
@@ -0,0 +1,13 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private interface Games.Runner : Object {
+	public abstract Gtk.Widget? get_display ();
```

Comment:
And allowing some classes to return `null` means that, every place where we call this method, we'll have to check whether the result is `null`.

See:
- [The null object pattern](https://en.wikipedia.org/wiki/Null_Object_pattern)
- [Null references: the billion dollar mistake](http://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare)